### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by markediez**
Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

**Comment by markediez** [comment]
My comment

**Comment by eddiesimeon** [comment]
anything

**Review by eddiesimeon — COMMENTED** [review]
looks like a test

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Argo Events - The Event-driven Workflow Automation Framework
-
+TEST
```
Comment (eddiesimeon) [inline README.md:2]: Very nice

**Review by eddiesimeon — COMMENTED** [review]
love it

**Comment by markediez** [comment]
@eddiesimeon did you get this

**Comment by eddiesimeon** [comment]
> @eddiesimeon did you get this

yes @markediez

**Comment by eddiesimeon** [comment]
test @eddiesimeon lol

**Comment by markediez** [comment]
is this not a comment

**Pull request closed without merging**